README.md

```diff
@@ -14,27 +14,24 @@ and add to your path.
 
 # Usage 
 
-Before usage, you should initialize. It sets up your github integration with your developer environments.
-It requires the following:
-- initialization file: [init-example.yaml](examples/init-example.yaml)
-- your person GitHub token to perform operations on your behalf. See more info [here](#GitHub-Access-Token)
+## Initialisztion
 
-**NOTE: Ensure that you update [init-example.yaml](examples/init-example.yaml) repositories to reference to your actual environments and software template repositories**.
+Before usage, you should initialise `corectl`. This will set up your GitHub integration with your developer environments.
+
+But first, you will need:
+  - A [GitHub personal token](#GitHub-Access-Token)
+  - To know which GitHub repository is your "environments repository". Please ask your administrator for this.
+
+To initialise `corectl`, run:
 
-To run initialization run:
 ```bash
 corectl config init
```

Comment:
instructions don't work, you get prompted for a file:
```
➜  corectl git:(main) corectl config init
Config not initialised, please run corectl config init.
Select file with configuration for initialization:
> ./ 
```

Reply:
@kkonstan I think you are using a different version of `corectl`. I removed the message "Config not initialised, please run corectl config init" when the command is `corectl config init`...

Reply:
of course I am, ignore, works as expected with latest build from main